src/pages/Meeting/DetailMeatingManagePage.tsx

```
@@ -0,0 +1,200 @@
+
+        <div className="flex flex-col lg:flex-row gap-10">
+          {/* 왼쪽: 조 생성 + 멤버 배정 */}
+          <div className="flex flex-col items-center gap-6 min-w-[401px]">
+            <h2 className="text-[#2C2C2C] text-[18px] font-medium self-start ml-2">
+              조 생성
+            </h2>
+
+            {groups.map((groupName, idx) => (
+              <div className="w-[401px] min-h-[310px] border-[2px] border-[#EAE5E2] bg-white rounded-[16px] px-4 py-3 text-[#2C2C2C] flex flex-col">
+                <div className = 'flex justify-between font-semibold text-lg mb-2'>
+                  <p className="">{groupName}</p>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)